### PR TITLE
Feature/fetch cms

### PR DESCRIPTION
**Pull request by yagotome**
## Fetch Content from CMS (in build time)

- Replace `gatsby-source-graphql` plugin with `gatsby-source-strapi`
- Query Home data for each language page
- Extract and map content come from CMS and use in HomePage

### CODE REVIEW

File: src/pages/index.en.js [hunk 1]
```diff
@@ -25,3 +14,84 @@ const IndexPage_en = () => {
 };
 
 export default IndexPage_en;
+
+// TODO: try to find a way of defining only one query for all languages, since [string interpolation isn't an option](https://github.com/gatsbyjs/gatsby/issues/3186)
```
Comment (BelfDev) [inline src/pages/index.en.js:18]: 👍

File: src/components/pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -15,17 +14,45 @@ import MethodologyCard from '../../common/cards/MethodologyCard';
...
+    citation: review.shortStatement,
+    organization: company.name,
+  }));
+
+  const staffMembersContent = useStrapiContent(staffMembers, { i18nFields: ['role', 'description'] }).map(({ name,  description, ...rest }) => ({
+    ...rest,
+    // TODO: rename `author` in StaffMember component to `name'
+    author: name,
+    bio: description,
+  }));
```
Comment (BelfDev) [inline src/components/pages/HomePage/HomePage.js:55]: I wish we could move this to the outer HomePage file to reduce the clutter, but I understand the current limitations (duplicate en/pt queries).

File: src/utils/strapi.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { useContext, useMemo } from 'react';
+import { cloneDeep, get, set } from 'lodash';
+
+import LanguageContext from '../contexts/LanguageContext';
+
+
+export const useStrapiContent = ({ edges }, { i18nFields = [] }) => {
```
Comment (BelfDev) [inline src/utils/strapi.js:7]: Smooth and optimized. 🚀

**Comment by BelfDev** [comment]
Please run `yarn format` :)

**Comment by yagotome** [comment]
> Nice job mate! I caught some warning you might want to take a look:
> 
> Screen Shot 2020-05-28 at 2 15 36 PM
> 
> Screen Shot 2020-05-28 at 2 16 15 PM
> 
> Screen Shot 2020-05-28 at 2 16 30 PM

Thank you! It seems these warns don't have side effect for us. Anyway, I'll be following this issue in Strapi plugin's repo: https://github.com/strapi/gatsby-source-strapi/issues/126

**Pull request merged**